sklearn/ensemble/_forest.py

```diff
@@ -1273,6 +1273,23 @@ class RandomForestClassifier(ForestClassifier):
 
         .. versionadded:: 0.22
 
+    monotonic_cst : array-like of int of shape (n_features), default=None
+        Indicates the monotonicity constraint to enforce on each feature.
```

Comment:
c21ea70cfcab53997b640c9ee3ec9fafc8c28d0d does that already.

Reply:
>  Also the link to the user guide for monotonicity constraints would be valuable as in HGBT.

Note that currently the `_monotonic_cst_gbdt` tag points to a subsection of the `_histogram_based_gradient_boosting` section, specific to HGBT. Are you suggesting to update this subsection to add that it also applies to DT, ET and RF + have docstrings in these classes point to it ? Or duplicate to all the relevant places in the user guide ? Or extract this to a common place ?

Reply:
I would just link from other tree based models contraints argument to the exact same place / link, pointing to the hgbt user guide, because it applies for the other trees, too.
We can later consider to restructure it.

Reply:
Alright then: 57c78a5fe

Reply:
Alright then: 57c78a5fe